directives/thebe.py

```diff
@@ -80,7 +80,13 @@ def init_thebe_core(app, env):
 
     # Add core libraries
     opts = {"async": "async", "data-aplus": "yes"}
+    css_opts = {"data-aplus": "yes"}
     app.add_js_file(filename="https://unpkg.com/thebelab@latest/lib/index.js", **opts)
+    app.add_js_file(filename="http://codemirror.net/mode/clike/clike.js", **opts)
```

Comment:
I assume you missed this one http URL when changing the others to https. I will just change this myself and merge.

Reply:
it seems i missed. thanks.